### DISCUSSION TITLE
Q-uploader - HTTP-response from factory prop

### DISCUSSION THREAD

**Discussion by bitmandk** [quasarframework/quasar#15119] [General - Components / Directives / etc]
I'm using a factory function with the q-uploader for setting formFields and authentication, like in the example below.

Each fileupload returns a HTTP response (JSON). How can I access the HTTPresponse - the factory function only returns the request. I have also tried to see if I can get some data in the uploaded event, but no data here either,

```
<script>
export default {
  methods: {
    factoryFn (file) {
      return new Promise((resolve, reject) => {
        // Retrieve JWT token from your store.
        const token = "myToken";
        resolve({
          url: 'http://localhost:4444/fileuploader/upload',
          method: 'POST',
          headers: [
            { name: 'Authorization', value: `Bearer ${token}` }
          ]
        })
      })
    }
  }
}
</script>
```
The HTTP response from the endpoint:

![image](https://user-images.githubusercontent.com/7941347/208075186-d77ee1db-ce27-4da4-b424-4e015659c7c9.png)

**Comment by bitmandk** [comment]
Updated the question with additional information

**Comment by createwithjames** [comment]
I'm having the exact same issue, did you ever find a resolution?
